protocols/v2/const-sv2/Cargo.toml

```
@@ -3,7 +3,7 @@ name = "const_sv2"
 version = "2.0.0"
 authors = ["fi3 <email@email.org>"]
 edition = "2018"
-description = "Sv2  constatnts"
+description = "SV2 constants"
```

After discussing, we want to continue using "Stratum V2" and "Sv2::

```suggestion
description = "Sv2 constants"
```